x/blob/client/cli/payforblob.go

```diff
@@ -32,55 +33,83 @@ const (
 
 func CmdPayForBlob() *cobra.Command {
 	cmd := &cobra.Command{
-		Use: "PayForBlobs namespaceID blob",
+		Use: "PayForBlobs [path/to/blob.json]",
 		// This example command can be run in a new terminal after running single-node.sh
-		Example: "celestia-appd tx blob PayForBlobs 0x00010203040506070809 0x48656c6c6f2c20576f726c6421 \\\n" +
+		Example: "celestia-appd tx blob PayForBlobs path/to/blob.json \\\n" +
 			"\t--chain-id private \\\n" +
 			"\t--from validator \\\n" +
 			"\t--keyring-backend test \\\n" +
 			"\t--fees 21000utia \\\n" +
 			"\t--yes",
-		Short: "Pay for a data blob to be published to Celestia.",
-		Long: "Pay for a data blob to be published to Celestia.\n" +
-			"namespaceID is the user-specifiable portion of a version 0 namespace. It must be a hex encoded string of 10 bytes.\n" +
-			"blob must be a hex encoded string of any length.\n" +
-			// TODO: allow for more than one blob to be sumbmitted via the CLI
-			"This command currently only supports a single blob per invocation.\n",
+		Short: "Pay for a data blobs to be published to Celestia.",
+		Long: "Pay for a data blobs to be published to Celestia.\n" +
+			`Where blob.json contains: 
+
+		{
+			"Blobs": [
+				{
+					"namespaceId": "0x00010203040506070809",
+					"blob": "0x48656c6c6f2c20576f726c6421"
+				},
+				{
+					"namespaceId": "0x00010203040506070809",
+					"blob": "0x48656c6c6f2c20576f726c6421"
+				}
+			]
+		}
+
+		namespaceID is the user-specifiable portion of a version 0 namespace. It must be a hex encoded string of 10 bytes.\n
+		blob must be a hex encoded string of any length.\n
+		
+		`,
 		Aliases: []string{"PayForBlob"},
 		Args: func(cmd *cobra.Command, args []string) error {
-			if len(args) < 2 {
-				return fmt.Errorf("PayForBlobs requires two arguments: namespaceID and blob")
+			if len(args) < 1 {
```

Comment:
[blocking] note this is still a breaking change to the CLI because the number of arguments has decreased. Is it possible to introduce this feature in a non-breaking way (i.e. via an optional flag)?

Reply:
hmmm a more simpler way is to add another cmd for this feature like how cosmos sdk did with their gov cmd, what do you think? With this aproach we dont have to modify the test too, only need to add 1 new testcase for the new cmd

Reply:
hmmm one option we can do is too move this new feature to a new cmd, that way old testcase wont be affect since we can just add a new one in. Other solution might be, we add an optional flag for the file path and disable the args number check if the flag is being use instead check for the file extension only ( because since we only need the flag value to submit blobs so the 2 args is useless by this point ). What do you think ?

Reply:
I would have a preference to keep this to a single command if possible but am also open to having two if we think it's cleaner. 

One option is that we just introduce a switch statement. If there is a single argument we treat it as a file, if it is two arguments we treat it as the namespace and blob

Reply:
@rootulp @cmwaters hey guys, i modified the code so it would handle both use cases ( single blob and multiple blobs )